app/controllers/file_configurations_controller.rb

```diff
@@ -20,9 +20,12 @@ def new
   def edit; end
 
   def download_template
-    send_data @file_configuration.column_attributes.values.to_csv,
-              filename: "template_file_configuration.csv",
-              type: "text/csv",
+    # This line ensures the CSV is read as UTF-8
+    bom = "\uFEFF"
+    csv_data = bom + @file_configuration.column_attributes.values.to_csv
+    send_data csv_data,
+              filename: "modele-#{@file_configuration.sheet_name.parameterize}.csv",
```

Comment:
niptick: je ne pense que le sheet_name apporte beaucoup d'informations par rapport au modèle ici.
Je pense que le nom de l'organisation est plus utile ici (bien que dans de rares cas il puisse y avoir plusieurs template par orga)

```suggestion
    send_data csv_data,
              filename: "modele-#{@organisation.name}.csv",
```